package-name-map/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "package-name-map",
+  "version": "0.1.0",
```

Comment:
I always start with "1.0.0-pre" myself

Reply:
changed to `1.0.0-pre.1` to be consistent with future prerelease bumps.